torch/types.py

```diff
@@ -12,30 +10,27 @@
     int as _int,
     str as _str,
 )
-from typing import Any, Dict, List, Sequence, Tuple, TYPE_CHECKING, Union
+from collections.abc import Sequence
+from typing import Any, TYPE_CHECKING, Union
 from typing_extensions import TypeAlias
 
 # `as` imports have better static analysis support than assignment `ExposedType: TypeAlias = HiddenType`
-from torch import (  # noqa: F401
+from torch import (
     device as _device,
-    DispatchKey as DispatchKey,
+    DispatchKey,
     dtype as _dtype,
-    layout as _layout,
-    qscheme as _qscheme,
-    Size as Size,
-    SymBool as SymBool,
-    SymFloat as SymFloat,
-    SymInt as SymInt,
-    Tensor as Tensor,
```

Comment:
I think they are intended to be re-exported.

Reply:
But the as clauses seem unnecessary.

Reply:
> But the as clauses seem unnecessary.

It is used to make `mypy` happy. The `as` clauses can be removed if you add them to `__all__`.

Reply:
Adding them to ``__all__`` is better.